src/components/Widget.tsx

```diff
@@ -75,32 +78,35 @@ export default function Widget(props: PropsWithChildren<WidgetProps>) {
   const [dialog, setDialog] = useState<HTMLDivElement | null>(props.dialog || null)
   return (
     <StrictMode>
-      <ThemeProvider theme={props.theme}>
-        <WidgetWrapper width={props.width} className={props.className}>
-          <I18nProvider locale={props.locale}>
-            <DialogWrapper ref={setDialog} />
-            <DialogProvider value={props.dialog || dialog}>
-              <ErrorBoundary onError={props.onError}>
-                <ReduxProvider store={store}>
-                  {
-                    // UI configuration must be passed to initial atom values, or the first frame will render incorrectly.
-                  }
-                  <AtomProvider initialValues={useInitialFlags(props as Flags)}>
-                    <WidgetUpdater {...props} />
-                    <Web3Provider {...(props as Web3Props)}>
-                      <BlockNumberProvider>
-                        <MulticallUpdater />
-                        <TransactionsUpdater {...(props as TransactionEventHandlers)} />
-                        <TokenListProvider list={props.tokenList}>{props.children}</TokenListProvider>
-                      </BlockNumberProvider>
-                    </Web3Provider>
-                  </AtomProvider>
-                </ReduxProvider>
-              </ErrorBoundary>
-            </DialogProvider>
-          </I18nProvider>
-        </WidgetWrapper>
-      </ThemeProvider>
+      <I18nProvider locale={props.locale}>
+        <DialogWrapper ref={setDialog} />
+        <DialogProvider value={props.dialog || dialog}>
+          <ErrorBoundary onError={props.onError}>
+            <ReduxProvider store={store}>
+              {
+                // UI configuration must be passed to initial atom values, or the first frame will render incorrectly.
+              }
+              <AtomProvider initialValues={useInitialFlags(props as Flags)}>
+                <WidgetUpdater {...props} />
+                <Web3Provider {...(props as Web3Props)}>
+                  <BlockNumberProvider>
+                    <ThemeProvider theme={props.theme}>
+                      <WidgetContainer width={props.width} className={props.className}>
+                        <WidgetWrapper>
```

Comment:
Needed to add container that didn't include background/border so that banner could be under widget rather than inside